scripts/unittest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+make -f elasticdl/Makefile
+
+# Go unittests
+pushd $GOPATH/src/elasticdl.org/elasticdl
+go install ./...
+go test ./...
```

Comment:
Add verbose and  coverage as:
`gotest -v -cover  ./... `

Reply:
Done